src/main/java/de/cismet/cids/custom/objecteditors/wunda_blau/EmobLadestationEditor.java

```diff
@@ -0,0 +1,2152 @@
+    public static final String FIELD__GEO_FIELD = "geo_field";                       // geom
+    public static final String FIELD__GEOREFERENZ__GEO_FIELD = "fk_geom.geo_field";  // emob_ladestation_geom
+    public static final String FIELD__FOTO = "foto";                                 // emob_ladestation
+    public static final String FIELD__STECKDOSE = "n_steckdosen";                    // emob_ladestation
+    public static final String FIELD__LADEKOSTEN = "fk_abrechnungsart";              // emob_ladestation
+    public static final String FIELD__LADEKOSTEN_NAME = "fk_abrechnungsart.name";    // emob_abrechnung_name
+    public static final String FIELD__STECKDOSE_LADESTATION = "fk_ladestation";      // emob_steckdose  
+    public static final String FIELD__STECKDOSE_SPANNUNG = "spannung";               // emob_steckdose  
+    public static final String FIELD__STECKDOSE_STROM = "strom";                     // emob_steckdose  
+    public static final String FIELD__STECKDOSE_LEISTUNG= "leistung";                // emob_steckdose  
+    public static final String FIELD__STECKDOSE_TYP = "fk_typ";                      // emob_steckdose  
+    public static final String FIELD__STECKDOSE_ANZAHL = "anzahl";                   // emob_steckdose     
+    public static final String TABLE_NAME = "emob_ladestation";     
+    public static final String TABLE_GEOM = "geom";                                  
+    public static final String TABLE_NAME_VERSATZ = "emob_versatz";                   
+    public static final String TABLE_NAME_STECKDOSE= "emob_steckdose";
+                       
+    public static final String TEXT_OPEN= "24 Stunden / 7 Tage";
+    
+    private static final String[] STECKDOSEN_COL_NAMES = new String[] { "kW", "A", "V", "Anz", "Typ" };
+    private static final String[] STECKDOSEN_PROP_NAMES = new String[] { "leistung", "strom", "spannung", "anzahl", "fk_typ" };
+    private static final Class[] STECKDOSEN_PROP_TYPES = new Class[] { Double.class, Integer.class, Integer.class, Integer.class, CidsBean.class};
+
+    //~ Instance fields --------------------------------------------------------
+
+    private boolean isEditor = true;
+    
+   
+    private final ImageIcon statusFalsch = new ImageIcon(
+            getClass().getResource("/de/cismet/cids/custom/objecteditors/wunda_blau/status-busy.png"));
+    private final ImageIcon statusOK = new ImageIcon(
+            getClass().getResource("/de/cismet/cids/custom/objecteditors/wunda_blau/status.png"));
+    
+    static {
+        final ConnectionContext connectionContext = ConnectionContext.create(
+                AbstractConnectionContext.Category.STATIC,
+                EmobLadestationEditor.class.getSimpleName());
+
+        STECKDOSENTYP_MC = ClassCacheMultiple.getMetaClass(
+                CidsBeanSupport.DOMAIN_NAME,
+                "emob_steckdosentyp",
+                connectionContext);
+    }
+    
```

Comment:
Unnötiger Static-Block (sollte vermieden werden wo es nur geht), der auch noch dazu führt, dass ein statischer ConnectionContext verwendet wird.

`STECKDOSENTYP_MC` bitte als normale Member-Variable deklarieren (der Name dann natürlich entsprechend in CamelCase-Notation), und in der `initWithConnectionContext` initialisieren.